### ISSUE TITLE
Operation Errors - cases parameter errors ('Equals' is required)

### ISSUE THREAD

**Issue by yuki-ACN** [Azure/logicapps#850]
### Describe the Bug

As you can infer from the error text, there is probably a problem with the validation of the switch action of the control connector on 'designer'.

This error occurs when trying to save changes on the 'designer'.
On the other hand, saving changes in the 'code view' does not occur it.
The workflow itself can also be executed normally.
In other words, we believe that the bug occurs only for changes made from the designer.

We did not see this bug until at least 8/10/2023 at midnight (JST); we just resumed work on 8/10/2023 at 9:00am(JST) and have encountered this bug.

### Plan Type

Standard

### Steps to Reproduce the Bug or Issue

As explained above.

### Workflow JSON

```json
Difficult to provide due to business critical workflow.
```

### Screenshots or Videos

<img width="902" alt="image" src="https://github.com/Azure/logicapps/assets/141885871/28d16c4b-1c24-4a26-96da-7408fc3daee7">
<img width="848" alt="image" src="https://github.com/Azure/logicapps/assets/141885871/29db91a5-0b4a-444a-89ed-2e6d83f44b27">

### Additional context

If anyone have other reports like this, please share the information.

**Comment by ChrisDRiley** [comment]
Noticing the exact same error myself, even just initializing a new variable causes this error for me and I am unable to save.
![logicapperror](https://github.com/Azure/logicapps/assets/24962393/3be08fbd-f755-486c-9b2c-a4a16b6c339b)

**Comment by SimonAdaptiv** [comment]
Yes, I noticed the same  on Friday. I saw the new Errors button after trying to save and thought that's pretty useful. Sadly, it just spun its wheels and the whole designer would no longer work i.e. I could no longer save. I worked around the issue by making changes in code view, which I don't need to say is very cumbersome.

**Comment by MalcolmKAdaptiv** [comment]
Editing a workflow with a case in VS Code  Designer or Code View also does not work, it puts 2 sequential actions as parallel actions.

**Comment by hartra344** [comment]
@Eric-B-Wu i think this is like related to your conditional validation. Can we get this hotfixed asap?

**Comment by tonylevaggi** [comment]
I am also seeing this issue in the Logic App Designer. Making a change in a step that isn't near the switch-case block errors with the same 'equals' is required behavior
